tests/e2e/specs/in-game/game-over/rankedMatches.spec.js

```diff
@@ -183,8 +182,9 @@ describe('Creating And Updating Ranked Matches', () => {
       .click();
     cy.get('#waiting-for-opponent-stalemate-scrim').should('be.visible');
     // Opponent confirms
-    cy.stalemateOpponent();
+    cy.acceptStalemateOpponent();
```

Comment:
Commands, like the app, now separate accepting stalemate from requesting one initially (though they resolve to the same endpoint in the legacy API)